fastlane_core/lib/fastlane_core/project.rb

```diff
@@ -172,6 +172,72 @@ def app_name
       return "App" # default value
     end
 
+    def build_for?
```

Comment:
true - not used, i initially thought i'd use this as a classifcation, but then it worked without, anyway i thought it might be nice to keep, or should be removed?